about.html

```diff
@@ -0,0 +1,173 @@
+            When you trust in our expertise to deliver you these extraordinary
+            experiences, it truly means the world to us.
+          </p>
+        </div>
+        <h4 class="about-hero__contact">
+          Please contact us on the email below for any further queries <br />
+          about our services.
+        </h4>
+        <a class="about-hero__email" href="#Contact"
+          >ashwincarvalho27@gmail.com</a
+        >
+      </div>
+    </section>
+    <section id="about-logo">
+      <div class="flex-container">
+        <h2>Air Sports Club Logo</h2>
+        <hr class="section__hr" />
+        <p>
+          The Air Sports Club logo was decided after a logo design competition
+          from 5th of June to 7th of July during the company logo contest, even
+          the friends of the Instructors took part and had great fun in the logo
+          contest.
+        </p>
+        <div class="about-logo__show">
+          <img
+            src="assets/images/air-sports-logo-1.png"
+            alt="Air Sports Club"
+          />
+        </div>
+      </div>
+    </section>
+    <section id="about-past">
+      <div class="flex-container">
+        <h2>Recent Sports Events</h2>
+        <hr class="section__hr" />
+        <p>
+          Take a look at the last two Sports Events which took place in Jan 2021
+          and Feb 2021
+        </p>
+        <div class="about-past__grid">
+          <article class="about-past__cards card1">
+            <h3>Jan 2019</h3>
+            <p>Events in Uttah</p>
+          </article>
+          <article class="about-past__cards card2">
+            <h3>Feb 2019</h3>
+            <p>Events in Brazil</p>
+          </article>
+        </div>
+      </div>
+    </section>
+    <section id="partners">
+      <div class="flex-container">
+        <h2>Partners</h2>
+        <hr class="section__hr" />
+        <div class="partner__flex">
+          <a href="" class="partner__links stumbleupon"
+            ><i class="fab fa-stumbleupon"></i> <br />
+            Stumble Upon</a
+          >
+          <a href="" class="partner__links airbnb"
+            ><i class="fab fa-airbnb"></i> <br />
+            Airbnb</a
+          >
+          <a href="" class="partner__links pied-piper"
+            ><i class="fab fa-pied-piper-alt"></i> <br />
+            Pied Piper</a
+          >
+          <a href="" class="partner__links soundcloud"
+            ><i class="fab fa-soundcloud"></i><br />
+            Sound Cloud</a
+          >
+          <a href="" class="partner__links ytravels"
+            ><i class="fab fa-yahoo"></i> <br />
+            Yahoo Travels</a
+          >
+        </div>
+      </div>
+    </section>
+    <section id="footer">
+      <div class="flex-container">
+        <img src="assets/images/air-sports-logo-1.png" alt="Air Sports Logo" />
+        <p>Copyright ©️ 2022 Air Sports Club. Some Rights Reserved</p>
+      </div>
+    </section>
```

Comment:
- [ ] As the previous reviewer mentioned, please change the `section` tag to a `footer` tag.